### PR TITLE
docs: `integrations/memory` consistency

### PR DESCRIPTION
**Pull request by leo-gan**
- updated titles and descriptions of the  `integrations/memory` notebooks into consistent and laconic format;
- removed `docs/extras/integrations/memory/motorhead_memory_managed.ipynb` file as a duplicate of the `docs/extras/integrations/memory/motorhead_memory.ipynb`;
- added `integrations/providers` Integration Cards for `dynamodb`, `motorhead`.
- updated `integrations/providers/redis.mdx` with links
- renamed several notebooks; updated `vercel.json` to reroute new names.

### CODE REVIEW

File: docs/docs_skeleton/vercel.json [hunk 1]
```diff
@@ -2509,9 +2509,21 @@
       "destination": "/docs/integrations/memory/cassandra_chat_message_history"
     },
     {
-      "source": "/docs/modules/memory/integrations/cassandra_chat_message_history",
+      "source": "/en/latest/modules/memory/examples/cassandra_chat_message_history.html",
```
Comment (baskaryan) [inline docs/docs_skeleton/vercel.json:2512]: why this change? isn't this duplicate of above

Comment (leo-gan) [reply docs/docs_skeleton/vercel.json:2512]: Oops. PyCharm automatically refactored it. My bad. I had to verify it anyway.

**Comment by leo-gan** [comment]
@baskaryan

**Comment by leo-gan** [comment]
@eyurtsev

**Comment by leo-gan** [comment]
@baskaryan Please, review it.

**Comment by leo-gan** [comment]
@baskaryan Thanks!

**Comment by leo-gan** [comment]
@baskaryan Could you, please, review it?

**Comment by leo-gan** [comment]
@hwchase17  Could you, please, review it?

**Comment by leo-gan** [comment]
@baskaryan Could you, please, review it?

**Comment by leo-gan** [comment]
@hwchase17 Could you, please, review it?
# testing this messaging. 95% probability that these messages are lost

**Review by hwchase17 — APPROVED** [review]
thank you!
